Source/Constraint.swift

```diff
@@ -362,7 +362,7 @@ public class Constraint {
         // special logic for updating
         if updateExisting {
             // get existing constraints for this view
-            let existingLayoutConstraints = reverse(layoutFrom!.snp_installedLayoutConstraints)
```

Comment:
@robertjpayne Quick question before I change this implementation (and split the PRs): What is the reasoning for `reverse` this array here?